src/main/java/mate/academy/service/OrderService.java

```diff
@@ -0,0 +1,20 @@
+package mate.academy.service;
+
+import java.util.List;
+import mate.academy.dto.order.OrderDto;
+import mate.academy.dto.order.item.OrderItemDto;
+import mate.academy.model.Order;
+import mate.academy.model.Status;
+import org.springframework.data.domain.Pageable;
+
+public interface OrderService {
+    OrderDto placeAnOrder(Long userId, Order order);
```

Comment:
This method should use `OrderRequestDto `instead of raw `Order `object.
```suggestion
    OrderDto placeOrder(Long userId, OrderRequestDto requestDto);
```